.travis.yml

```diff
@@ -1,13 +1,17 @@
 branches:
   only:
-    - master 
+    - master
+git:
+  depth: 1
 language: node_js
 node_js:
   - node
 os: linux
 
 script:
-  - npm install tools/pkgeval_badges
```

Comment:
Why was this change needed?

Reply:
Not sure, but the build wouldn't pass until I made those changes.